[0.000000] (-) TimerEvent: {}
[0.000552] (-) JobUnselected: {'identifier': 'hardware'}
[0.000663] (-) JobUnselected: {'identifier': 'my_mas'}
[0.000733] (py_pubsub) JobQueued: {'identifier': 'py_pubsub', 'dependencies': OrderedDict([('my_mas', '/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/install/my_mas')])}
[0.001875] (py_pubsub) JobStarted: {'identifier': 'py_pubsub'}
[0.099370] (-) TimerEvent: {}
[0.200467] (-) TimerEvent: {}
[0.301621] (-) TimerEvent: {}
[0.402153] (-) TimerEvent: {}
[0.504338] (-) TimerEvent: {}
[0.606251] (-) TimerEvent: {}
[0.711564] (-) TimerEvent: {}
[0.812176] (-) TimerEvent: {}
[0.912768] (-) TimerEvent: {}
[1.013343] (-) TimerEvent: {}
[1.113984] (-) TimerEvent: {}
[1.214630] (-) TimerEvent: {}
[1.315143] (-) TimerEvent: {}
[1.415699] (-) TimerEvent: {}
[1.516324] (-) TimerEvent: {}
[1.616857] (-) TimerEvent: {}
[1.622163] (py_pubsub) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 51, in build\n    setup_py_data = get_setup_data(self.context.pkg, env)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/__init__.py", line 20, in get_setup_data\n    return dict(pkg.metadata[key](env))\n  File "/usr/lib/python3/dist-packages/colcon_ros/package_augmentation/ros_ament_python.py", line 57, in getter\n    return get_setup_information(\n  File "/usr/lib/python3/dist-packages/colcon_python_setup_py/package_identification/python_setup_py.py", line 241, in get_setup_information\n    _setup_information_cache[hashable_env] = _get_setup_information(\n  File "/usr/lib/python3/dist-packages/colcon_python_setup_py/package_identification/python_setup_py.py", line 288, in _get_setup_information\n    result = subprocess.run(\n  File "/usr/lib/python3.10/subprocess.py", line 526, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError: Command \'[\'/usr/bin/python3\', \'-c\', \'import sys;from contextlib import suppress;exec("with suppress(ImportError):    from setuptools.extern.packaging.specifiers    import SpecifierSet");exec("with suppress(ImportError):    from packaging.specifiers import SpecifierSet");from distutils.core import run_setup;dist = run_setup(    \\\'setup.py\\\', script_args=(\\\'--dry-run\\\',), stop_after=\\\'config\\\');skip_keys = (\\\'cmdclass\\\', \\\'distclass\\\', \\\'ext_modules\\\', \\\'metadata\\\');data = {    key: value for key, value in dist.__dict__.items()     if (        not key.startswith(\\\'_\\\') and         not callable(value) and         key not in skip_keys and         key not in dist.display_option_names    )};data[\\\'metadata\\\'] = {    k: v for k, v in dist.metadata.__dict__.items()     if k not in (\\\'license_files\\\', \\\'provides_extras\\\')};sys.stdout.buffer.write(repr(data).encode(\\\'utf-8\\\'))\']\' returned non-zero exit status 1.\n'}
[1.624516] (py_pubsub) JobEnded: {'identifier': 'py_pubsub', 'rc': 1}
[1.633220] (-) EventReactorShutdown: {}
